app/src/main/java/com/github/codetanzania/ui/fragment/IssueDetailsFragment.java

```diff
@@ -1,110 +1,116 @@
+
+    // reference to the recycler view. used to show map and image captured when
+    // user submitted an issue
+    // private RecyclerView mAttachmentsRecyclerView;
+
+    @Override
+    public View onCreateView(
+            LayoutInflater inflater, ViewGroup group, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.frag_issue_details, group, false);
+    }
+
+    public static IssueDetailsFragment getInstance(Bundle args) {
+        IssueDetailsFragment inst = new IssueDetailsFragment();
+        inst.setArguments(args);
+        return inst;
+    }
+
+    @Override
+    public void onViewCreated(View fragView, Bundle savedInstanceState) {
+
+        Bundle args = getArguments();
+        ServiceRequest serviceRequest = args.getParcelable(Constants.Const.TICKET);
+        assert serviceRequest != null;
+
+        int numFrags = serviceRequest.attachments == null || serviceRequest.attachments.isEmpty() ? 1 : 2;
+
+        // debug
+        Log.d(TAG, String.format("%d", numFrags));
+
+        // view pager
+        ViewPager viewPager = (ViewPager) fragView.findViewById(R.id.viewPager);
+        IssueItemsViewPagerAdapter viewPagerAdapter = new IssueItemsViewPagerAdapter(
+                getChildFragmentManager(), serviceRequest, numFrags);
+        viewPager.setAdapter(viewPagerAdapter);
+        CircleIndicator indicator = (CircleIndicator) fragView.findViewById(R.id.indicator);
+        indicator.setViewPager(viewPager);
+
+        // bind description data
+        TextView tvIssueDate = (TextView) fragView.findViewById(R.id.tv_IssueDate);
+        tvIssueDate.setText(Util.formatDate(serviceRequest.createdAt, "yyyy-MM-dd HH:mm:ss"));
+
+        // tvReportTimestamp.setText(timestamp);
+        TextView tvIssueCategoryContent = (TextView) fragView.findViewById(R.id.tv_IssueCategoryContent);
+        tvIssueCategoryContent.setText(serviceRequest.service.name);
+        TextView tvIssueContent = (TextView) fragView.findViewById(R.id.tv_IssueContent);
+        tvIssueContent.setText(serviceRequest.description);
+
+        List<Comment> comments = new ArrayList<>();
+        Comment comment;
+
+        TextView tvIssueStatus = (TextView) fragView.findViewById(R.id.tv_IssueStatus);
+
+        // special case: when issue is resolved
+        if (serviceRequest.resolvedAt != null) {
+            comment = new Comment();
+            comment.commentor = serviceRequest.reporter.name;
+            comment.content = getString(R.string.text_close_status);
+            comment.timestamp = serviceRequest.resolvedAt;
+            comments.add(comment);
+
+            tvIssueStatus.setText(
+                    String.format(Locale.getDefault(), "Closed after %s",
+                            Util.timeElapse(serviceRequest.createdAt, serviceRequest.resolvedAt, getActivity())));
+        } else {
+            tvIssueStatus.setText(R.string.text_pending_description);
+        }
```

Comment:
Add todo: this will be refactored in the future when Lally begins to send comments from the backend.